src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -1056,61 +1057,57 @@ private static ulong NumberToDoubleFloatingPointBits(ref NumberBuffer number, in
             uint integerDigitsPresent = Math.Min(positiveExponent, totalDigits);
             uint fractionalDigitsPresent = totalDigits - integerDigitsPresent;
 
-            // When the number of significant digits is less than or equal to MaxMantissaFastPath and the
```

Comment:
Changes are not as huge as they look like .... it's a matter of indentation (with new condition for  digits <=19) & moving a comment section ....
![image](https://user-images.githubusercontent.com/25437410/144712545-2d2ed526-e4ea-4fc2-9484-c120d79a301c.png)